### DISCUSSION TITLE
Content not appearing after changing viewUrl

### DISCUSSION THREAD

**Discussion by alexandra-simeonova** [SAP/luigi#2645] [Q&A] [answered]
When I edit the viewUrl in my Luigi config, the page appears for a split second, and then I see a loading indicator/three dots appear in the content area. How do I resolve this?

**Comment by stanleychh** [comment, marked as answer]
This happens if the micro frontend app does not use Luigi Client. (See the second FAQ question here https://docs.luigi-project.io/docs/faq?section=ui-questions). You can either add Luigi Client to your micro frontend, or disable the loading indicator for your navigation node in the luigi-config.js, for example:
```
{ 
    pathSegment: 'one', 
    label: 'Child 1', 
    category:  'Section One,
    loadingIndicator: {
        enabled: false
    },
    viewUrl: 'https://example.com'
}
```
